package.json

```diff
@@ -52,6 +52,7 @@
     "d3-collection": "^1.0.7",
     "dayjs": "^1.8.21",
     "extract-css-chunks-webpack-plugin": "^4.7.4",
+    "hoist-non-react-statics": "^3.3.2",
```

Comment:
Why do you need this?

Reply:
When a component is wrapped within HOC component like [this for example](https://github.com/oacore/dashboard/blob/master/utils/withErrorBoundary.jsx#L5). All static properties are lost.

Concretely I needed it for using `<Table.Sidebar />` because table component uses its own error boundary.

Reply:
I consider this being workaround and do not recommend to do so.

Reply:
If you can find a better solution do it. This is my way how I fixed. That library is downloaded almost 9M times per week. I don't consider this as a workaround.

Reply:
It's even recommended by React docs
https://reactjs.org/docs/higher-order-components.html#static-methods-must-be-copied-over